### PR TITLE
Raster Legend for Palette-Colorizer

### PR DESCRIPTION
**Pull request by KaroFr**

### CODE REVIEW

**Comment by KaroFr** [comment]
[Raster-Legende für Palette-Colorizer](https://trello.com/c/c7CntkrR/382-raster-legende-f%C3%BCr-palette-colorizer)

File: projects/wave-core-new/src/lib/layers/legend/legend-raster/raster-legend.component.html [hunk 1]
```diff
@@ -12,6 +12,15 @@
         </tr>
     </ng-template>
 
+    <ng-template [ngIf]="layer.symbology.colorizer.isDiscrete()">
+        <tr *ngFor="let breakpoint of layer.symbology.colorizer.getBreakpoints(); let isFirst = first">
```
Comment (michaelmattig) [inline projects/wave-core-new/src/lib/layers/legend/legend-raster/raster-legend.component.html:16]: if you don't use the `isFirst` variable you can remove it

**Review by michaelmattig — CHANGES_REQUESTED** [review]

File: projects/wave-core-new/src/lib/layers/legend/legend-raster/raster-legend.component.html (hunk 1 above)
Comment (KaroFr) [reply projects/wave-core-new/src/lib/layers/legend/legend-raster/raster-legend.component.html]: I removed it e17e0f5a4797d018cde265ee8cd07d444dbe25e6

**Review by michaelmattig — APPROVED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**